src/commands/migrate.js

```diff
@@ -35,7 +35,9 @@ Starts a migration to move users between different versions of your app. You may
 
 Only migrate users between non-breaking versions, use \`zapier deprecate\` if you have breaking changes!
 
-Migrations can take between 5-10 minutes, so be patient and check \`zapier history\` to track the status.
+Migrations can take between 5-10 minutes, so be patient and check \`zapier history\` to track the status
+
+This process should be transparent to users, as they aren't emailed about this update/migration, and if there are no breaking changes, the Zaps should continue to work as they were.
```

Comment:
I'd say break this up, or remove the extra `if there are no breaking changes` since Line 36 already explains this. 

Maybe as simple as:

`Note: since a migration is only for non-breaking changes, users are not emailed about the update/migration.`